### ISSUE TITLE
Testing for numeric data

### ISSUE THREAD

**Issue by paulstey** [carloscinelli/benford.analysis#5]
Hello Carlos, 

When you are testing whether or not the data supplied to `extract.digits()` is numeric data, I think you should be using `is.numeric()`. As it stands now, the use of `class(data) != "numeric"` produces unexpected errors when the class is `"integer"`. Consider the following:

``` R
library(benford.analysis)

dat <- data.frame(v1 = 1:5, v2 = c(1, 2, 3, 4, 5))

benford(dat$v1)          # produces error
```

This error is returned because while `dat$v1` is clearly numeric, the call `class(dat$v1)` will return `"integer"`.

On a different note, thank you for a wonderful R package!

-Paul

**Comment by carloscinelli** [comment]
You are right, Paul, thanks!

**Comment by carloscinelli** [comment]
Paul, I have just corrected the issue, the github version should be running fine now!
